src/languages/php-template.js

```diff
@@ -0,0 +1,30 @@
+/*
+Language: PHP Template
+Requires: xml.js, php.js
+Author: Josh Goebel <hello@joshgoebel.com>
+Website: https://www.php.net
+Category: common
+*/
+
+export default function(hljs) {
+  return {
+    name: "PHP template",
+    subLanguage: 'xml',
```

Comment:
I can't say I like this two-grammars approach very much...
But looking at your PR I got a thought: what if we allow two sublanguages here? I. e.
```js
    subLanguage: ['xml','php'],
```
This way we got one grammar that resolves into either `xml` + contained `php`, or just `php`.
If only we had a way to disallow `php` to act as a top-level language...

Reply:
> If only we had a way to disallow php to act as a top-level language...

But it IS a top-level language... someone could post a snippet of PHP without the tags... in which case it's a php snippet, ie `php` code... meanwhile a `php-template` is HTML+PHP...

> what if we allow two sublanguages here?

That makes me head hurt - and what is the advantage of doing it that way?  PHP code on it's own is NOT a php template.

Reply:
> I can't say I like this two-grammars approach very much...

It's consistent with how we do VBScript-HTML and how I've suggested we do other templating languages.  For these type of things you have 3 things happening:

- The language (say Ruby)
- The templating language (say ERB)
- The actual output format (say HTML)

Those would be 3 different grammars.  This is true for PHP also... the language itself is technically distinct from the templating language.

Reply:
Just rename `php` to `_php`, and `php-template` to `php`.
So that you can use single `php` for both php snippets and php templates.

Reply:
How would that work for ERB or VBscript?  I'm trying to also look at the BIG picture here on how we handling templating in general.

Reply:
> So that you can use single php for both php snippets and php templates.

REMEMBER, no one is using `php` now for PHP... well they might be, but they're more likely using `xml` since that highlights HTML + PHP.

Reply:
> REMEMBER, no one is using php now for PHP... well they might be, but they're more likely using xml since that highlights HTML + PHP.

Right. And we consider it bad, because it makes xml grammar bloated.
On the other hand, adding xml support to the core language doesn't seem bad. Or it does?

> How would that work for ERB or VBscript?

The same way.
(leaving aside existing ERB and VBScript-html for the sake of discussion)